### DISCUSSION TITLE
How to disable coverage metrics in console and only output to file.

### DISCUSSION THREAD

**Discussion by SamHuoJudiAi** [coverlet-coverage/coverlet#1346] [Q&A] [answered]
Hi,

I am using the following nuget package versions:

- coverlet.msbuild: v2.9.0
- Microsoft.CodeCoverage: v16.9.1

I am trying to figure out how to disable cover metrics in the console when I run dotnet test but still output the file.  The reason why I am trying to do this is to reduce the information that is being captured on the console when we run Test/Coverage on GitHub Actions.

Thanks!

**Comment by MarcoRossignoli** [comment, marked as answer]
You should not use both coverage package. You should or coverlet or Microsoft.CodeCoverage they're doing the same thing and by design `dotnet test` doesn't collect coverage.
Can you open a ticket here https://github.com/microsoft/vstest/issues it's unrelated to coverlet.

**Reply by SamHuoJudiAi** [reply]
Thanks for your reply.
